### DISCUSSION TITLE
commandline crop parameters after openslideload - other than autocrop?

### DISCUSSION THREAD

**Discussion by kfisher00** [libvips/libvips#2901] [Help wanted]
Hello, I have a scanned slide image (mrxs) that exceeds conventional TIFF and apparently BigTIFF XY dimensions. I was hoping to load file via openslideload command line, then pass four coordinates so as to subcrop and export the image. I only have found autocrop run-time flag, am I overlooking some documentation, or this is going to be a script/routine based export?

Thanks for your thoughts/advice in advance.

**Comment by jcupitt** [comment]
Hi @kfisher00,

What do you see with `vipsheader`? For example I see:

```
$ vipsheader Mirax2.2-1.mrxs 
Mirax2.2-1.mrxs: 101832x219976 uchar, 4 bands, srgb, openslideload
```

Unfortunately, the mrxs format has "evolved" recently and it has broken everyone's software. With a recent file I see:

```
$ vipsheader 026-UK\ Koeln\ LMHO\ 4690_12.mrxs 
vipsheader: openslide2vips: opening slide: Cannot read slide position info: Expected 1 value
```

If you have one of these mrxs files, you are in trouble. There are patches which try to fix this, but you'll need to do some development work.

**Reply by jcupitt** [reply]
To answer your question, your can crop any file with eg.:

```
vips crop something.mrxs x.jpg 10000 200 1000 1000
```

Where the four numbers are left, top, width, height.
